interface/modules/custom_modules/oe-module-payroll/moduleConfig.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/*
+ * Provider Payroll
+ *
+ * @package OpenEMR
+ * @link    https://www.open-emr.org
+ * author Sherwin Gaddis <sherwingaddis@gmail.com>
+ * All Rights Reserved
+ * @copyright Copyright (c) 2023.
+ */
+
+$sessionAllowWrite = true;
+require_once(__DIR__ . "/../../../globals.php");
+$module_config = 1;
+?>
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+<div class="container">
+    <div class="row">
+        <div class="col-md-12">
+            <h2><?php echo xlt("Provider Payroll") ?></h2>
+            <p><?php echo xlt("This module allows you to view and update the payroll information for each provider.") ?></p>
+            <h2><?php echo xlt("Instructions") ?></h2>
+            <p><?php echo xlt("Now that the modules has been actived, go to the main menu and select Admin. Under
+            Admin, there should be a selection for 'Set Providers Rate'. Once that is selected, you will see the form to
+            enter the rate for each provider.
+            ") ?></p>
```

Comment:
translations constants need to be on one line or it will not be correctly processed/translated in OpenEMR's translation pipeline/engine  (same issue also for the 3 translation constants below)